modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/RowVersion.java

```diff
@@ -38,10 +41,24 @@ public class RowVersion extends StorableBase {
      */
     public static final long NULL_LINK = 0;
 
+    private static final int TIMESTAMP_STORE_SIZE_BYTES = 2 * Long.BYTES;
+    private static final int NEXT_LINK_STORE_SIZE_BYTES = PartitionlessLinks.PARTITIONLESS_LINK_SIZE_BYTES;
+    private static final int VALUE_SIZE_STORE_SIZE_BYTES = Integer.BYTES;
+
+    public static final int TIMESTAMP_OFFSET = 0;
+    public static final int NEXT_LINK_OFFSET = TIMESTAMP_STORE_SIZE_BYTES;
+    public static final int VALUE_SIZE_OFFSET = NEXT_LINK_OFFSET + NEXT_LINK_STORE_SIZE_BYTES;
+    public static final int VALUE_OFFSET = VALUE_SIZE_OFFSET + VALUE_SIZE_STORE_SIZE_BYTES;
+
+    private final int partitionId;
+    private long link;
+
     @Nullable
     private final Timestamp timestamp;
     private final long nextLink;
     private final int valueSize;
+    @IgniteToStringExclude
```

Comment:
I know that you are against these comments, but please use line separators more often, with these changes code becomes more and more inconsistent.